app/common/renderer/lib/vendor/bitbar.js

```diff
@@ -4,30 +4,26 @@ export class BitbarVendor extends BaseVendor {
   /**
    * @override
    */
-  async apply() {
+  async setProperties() {
```

Comment:
I find it confusing that there are two methods saveProperties and setProperties. Perhaps they could have more descriptive names